<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>31</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qlineargradient(spread:pad, x1:0.542249, y1:1, x2:0.527, y2:0, stop:0 rgba(68, 0, 127, 255), stop:0.502488 rgba(170, 71, 137, 255), stop:1 rgba(255, 131, 145, 255));

border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>621</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="verUsuarios_tab">
     <attribute name="title">
      <string>Editar Datos</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Usuario actual:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contra_lineedit_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: rgba(46,82,101,200);
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmar_boton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#confirmar_boton_2{
	background-color:rgba(68, 0, 127, 255);
	color: white;
	border-radius: 5px;
}

QPushButton#confirmar_boton_2:pressed{
	padding-left: 5px;
	padding-top:5px;
	background-color: rgba(255, 131, 145, 255);
 	background-position: calc(100% - 10px)center;
}

QPushButton#confirmar_boton_2:hover{
	background-color:rgba(255, 131, 145, 255);
}</string>
      </property>
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>By: Yael Salinas</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Cambiar usuario:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contra_lineedit_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: rgba(46,82,101,200);
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Cambiar contraseña:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contra_lineedit_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>300</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: rgba(46,82,101,200);
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Usuario actual:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contra_lineedit_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>250</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: rgba(46,82,101,200);
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmar_boton_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#confirmar_boton_3{
	background-color:rgba(68, 0, 127, 255);
	color: white;
	border-radius: 5px;
}

QPushButton#confirmar_boton_3:pressed{
	padding-left: 5px;
	padding-top:5px;
	background-color: rgba(255, 131, 145, 255);
 	background-position: calc(100% - 10px)center;
}

QPushButton#confirmar_boton_3:hover{
	background-color:rgba(255, 131, 145, 255);
}</string>
      </property>
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
     <widget class="QLabel" name="aviso_label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="aviso_label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>340</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Comprar</string>
     </attribute>
     <widget class="QTableView" name="buscar_tableview">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>521</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Búsqueda:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>By: Yael Salinas</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="buscar_lineedit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: rgba(46,82,101,200);
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Nombre del artículo:&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Cantidad:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombre_comprar">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: rgba(46,82,101,200);
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cantidad_comprar">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: rgba(46,82,101,200);
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmar_compra">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#confirmar_compra{
	background-color:rgba(68, 0, 127, 255);
	color: white;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>Comprar</string>
      </property>
     </widget>
     <widget class="QLabel" name="aviso_compra">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="aviso_compra_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>370</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Historial</string>
     </attribute>
     <widget class="QTableView" name="ver_historial">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>521</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="salirButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 131, 145, 255);</string>
    </property>
    <property name="text">
     <string>❌​</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 131, 145, 255);</string>
    </property>
    <property name="text">
     <string>🔃​</string>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>label</zorder>
   <zorder>salirButton</zorder>
   <zorder>refresh_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
